### ISSUE TITLE
Add support for escaping the backquote in vertex statements

### ISSUE THREAD

**Issue by CrazyPython** [r2d4/virgo#5]
This should be permitted:
```
a -> b
a = `\``
```

**Comment by r2d4** [comment]
Agreed. Maybe we should adopt the Go syntax with `""` for interpreted strings and `` for raw string literals

**Comment by CrazyPython** [comment]
`""` would be great.

Also, can you point me to where you implement Topological Sort in your Go implementation? I'm almost done with my JS implementation, it just needs TopoSort.

**Comment by r2d4** [comment]
https://github.com/r2d4/virgo/blob/default/pkg/virgo/topsort.go
